### ISSUE TITLE
dyld: lazy symbol binding failed: Symbol not found: _prepare_multi_stmt

### ISSUE THREAD

**Issue by ttilberg** [digital-fabric/extralite#3]
Hello! I just helped make an ETL app that used a bunch of threaded queries, not realizing there would be GVL (since it's IO). I read about this gem in this week's Ruby Weekly and was excited for the prospect!

When trying to use this from my mac, and another developer's mac, we get the following error:

```
[2] pry(main)> DB = Sequel.connect('extralite:blog.db')
dyld: lazy symbol binding failed: Symbol not found: _prepare_multi_stmt
  Referenced from: /Users/ttilberg/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/extralite-1.10/lib/extralite_ext.bundle
  Expected in: flat namespace

dyld: Symbol not found: _prepare_multi_stmt
  Referenced from: /Users/ttilberg/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/extralite-1.10/lib/extralite_ext.bundle
  Expected in: flat namespace

[1]    62097 abort      pry
```

Are there some c dependencies we need to make sure are installed? Can we add it to the readme, and better yet, is there a way to present a better error message?

**Comment by ciconia** [comment]
Thanks for reporting this. Fixed in version 1.11.

**Comment by ttilberg** [comment]
Thanks for the zippy fix, and for sharing your work!
